docs/conversion_examples_gallery/conversion_example_gallery.rst

```diff
@@ -81,6 +81,7 @@ Behavior
     DeepLabCut <behavior/deeplabcut>
     SLEAP <behavior/sleap>
     Videos <behavior/video>
+    Audio <behavior/audio>
```

Comment:
can be removed

Reply:
I think it wasn't showing up when I didn't do that, but I'll double check.